### PR TITLE
Tweak construction prices and times.

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request

Railings cost  4 → 2 sheets of metal
Catwalk cost 3 → 2 steel rods, speed adjusted from *six* seconds to 2 at 0 MEC and instant at 40 MEC
Underplating→Plating cost 2 → 1 sheet of metal, speed adjusted from 0.5 seconds to 3 at 0 MEC and instant at 40 MEC
Lattice→Underplating cost 4 → 1 sheet of metal, speed adjusted from 8 seconds to 4 at 0 MEC and instant at 40 MEC
Standartized (kinda?) construction on exoplanets/open space/space tiles
Most stuff that goes onto walls (air/fire alarms, light fixtures) craft time lowered from 9 to 6 seconds default (it is affected by MEC and other things too)

There are unrelated changes in this PR (indentation fixes) because compiler wouldn't shut up about them otherwise

## Why It's Good For The Game

it's no longer CBT

## Changelog
:cl:
balance: Railings cost 4 -> 2 sheets of metal
balance: Catwalk cost 3 -> 2 steel rods, speed adjusted from six seconds to 2 at 0 MEC and instant at 40 MEC
balance:  Underplating -> Plating cost 2 -> 1 sheet of metal, speed adjusted from 0.5 seconds to 3 at 0 MEC and instant at 40 MEC
balance:  Lattice -> Underplating cost 4 ->1 sheet of metal, speed adjusted from 8 seconds to 4 at 0 MEC and instant at 40 MEC
balance: most wall frame recipes default construction time is 6 seconds instead 9 seconds
/:cl:

### CODE REVIEW

**Comment by Firefox13** [comment]
I'm pretty sure there's a lot more things that should be sped up so feel free to suggest

**Comment by garrulousTeamotei** [comment]
Light frame fixtures feel like they need a buff to build time.

**Comment by Firefox13** [comment]
I changed their speed to `WORKTIME_FAST` (6 seconds vs. older 9)

**Review by hyperioo — APPROVED** [review]

File: code/game/turfs/space/space.dm [hunk 1]
```diff
@@ -57,39 +56,24 @@
...
 		if (!mat.name == MATERIAL_STEEL)
 			return
+
 		var/obj/structure/lattice/L = locate(/obj/structure/lattice, src)
 		if(L)
-			var/obj/item/stack/tile/S = C
-			if (S.get_amount() < 1)
-				return
-			qdel(L)
+			var/obj/item/stack/tile/floor/S = C
```
Comment (hyperioo) [inline code/game/turfs/space/space.dm:67]: Why do you cast C as a `obj/item/stack/tile/floor` when it's actually a `/obj/item/stack/material` (as checked before in the if)

Comment (TheShown911) [reply code/game/turfs/space/space.dm:67]: Suspicious.

Comment (Firefox13) [reply code/game/turfs/space/space.dm:67]: I didn't really thought about this part when adjusting the parts, but I believe this is a remnant of ability to use tiles on lattices.
Let me remove it in a moment.

Comment (Firefox13) [reply code/game/turfs/space/space.dm]: I edited out those lines.
It's really odd that this code worked.

**Pull request merged**